forms.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="bg">
+    <head>
+        <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" >
+        <title> Форми </title>
+    </head>
+    <body>
+     <form method="post" action="forms.php">
+     <fieldset>
+     <legend> Информация за избираемата дисциплина </legend>
+        <label for="courseTitle">Име на курса:</label>
+         <input id="coursetitle" name="title" require> </br>
+        <label for ="teacherName">Име на преподавателя:</label>
+         <input id="teacherName" name="name" require ></br>
+        <label for ="description">Описание на курса:</label>
+         <input id="description" name="description" require ></br>
+        <label for ="group">Група:</label>
+        <select id="group" name="group">
+            <option value="М">М </option>
+            <option value="ПМ">ПМ </option>
+            <option value="ОКН">ОКН </option>
+            <option value="ЯКН">ЯКН </option>
+            </select></br>
+            <label for ="credits">Кредити:</label>
+        <input id="credits" name="credits"></br>
```

Comment:
I am allowed to enter text into this input and I shouldn't be able to. Please change the input type. Also please limit it so it cannot be bigger than some value or less than 0 (through html).

Reply:
This way the user will know what values he is allowed to use even before clicking on submit. And we want to keep users happy. ^.^